duckduckgo/macos-browser
# Drop unwanted titles from Imported Logins

Task/Issue URL: https://app.asana.com/0/1204099484721401/1204318806095418/f

**Description**:
When importing logins, remove the title when it matches the URL or the Title matches a specific pattern.


**Steps to test this PR**:
No need for manual testing.  Changes have no UI and are covered by a new unit test, but you can import a few logins with titles matching the following patterns and the host as URL, and check all are removed:

- duck.com
- duck.com (test@duck.com)
- signin.duck.com
- signin.duck.com (test@duck.com.co)
- https://signin.duck.com
- https://signin.duck.com (test@duck.com.co)
- https://signin.duck.com?page.php?test=variable1&b=variable2
- https://signin.duck.com/section/page.php?test=variable1&b=variable2 

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
